src/youtubeStream.py

```diff
@@ -0,0 +1,123 @@
+import time
+import cv2
+from yt_dlp import YoutubeDL
+
+from typing import ClassVar, Dict, Mapping, Optional, Any, Tuple, List
+from viam.media.video import ViamImage, Optional, CameraMimeType, NamedImage
+from typing_extensions import Self
+from viam.proto.common import ResponseMetadata
```

Comment:
Optional nit: 
1. alphabetical import ordering 
1. Standard library, 3rd party libraries, and viam imports should be in their own blocks